Queueomatic/Server/Endpoints/User/Delete/DeleteUserEndpoint.cs

```diff
@@ -15,11 +15,19 @@ public DeleteUserEndpoint(IUnitOfWork unitOfWork)
     public override void Configure()
     {
         Delete("/users/{email}");
+        Roles("User");
         Policies("SignedInUser");
     }
 
     public override async Task HandleAsync(DeleteUserRequest req, CancellationToken ct)
     {
+
+        if (!User.IsInRole("User") && !req.Email.Equals(req.UserId))
```

Comment:
If only a user can access this endpoint, then why do you need to check if the user has that role?

That is already done in the policy.